qiskit/providers/__init__.py

```diff
@@ -184,37 +184,60 @@ def backends(self, name=None, **kwargs):
 provide the interface between Qiskit and the hardware or simulator that will
 execute circuits. This includes providing the necessary information to
 describe a backend to the compiler so that it can embed and optimize any
-circuit for the backend. There are 3 required things in every backend object: a
-backend configuration property, a :obj:`~qiskit.providers.BackendV1.run` method, and a
-:obj:`~qiskit.providers.BackendV1._default_options` method. For example, a
-minimum working example would be something like::
-
-    from qiskit.providers import BackendV1 as Backend
-    from qiskit.providers.models import BackendConfiguration
+circuit for the backend. There are 4 required things in every backend object: a
+:attr:`~qiskit.providers.BackendV2.target` property to define the model of the
+backend for the compiler, a :attr:`~qiskit.providers.BackendV2.max_circuits`
+property to define a limit on the number of circuits the backend can execute in
+a single batch job (if there is no limit ``None`` can be used), a
+:meth:`~qiskit.providers.BackendV2.run` method to accept job submissions, and
+a :obj:`~qiskit.providers.BackendV2._default_options` method to define the
+user configurable options and their default values. For example, a minimum working
+example would be something like::
+
+    from qiskit.providers import BackendV2 as Backend
+    from qiskit.transpiler import Target
     from qiskit.providers import Options
+    from qiskit.circuit import Parameter, Measure
+    from qiskit.circuit.library import PhaseGate, SXGate, UGate, CXGate, IGate
 
 
     class Mybackend(Backend):
 
-        configuration = {
-            'backend_name': 'my_backend',
-            'backend_version': '1.0.0',
-            'n_qubits': 5,
-            'basis_gates': ['p', 'sx', 'u', 'cx', 'id'],
-            'coupling_map': [[0, 1], [1, 2], [2, 3], [3, 4]],
-            'simulator': False,
-            'local': False,
-            'conditional': True,
-            'open_pulse': False,
-            'memory': True,
-            'max_shots': 8196,
-            'gates': []
-        }
+        def __init(self):
```

Comment:
`__init__`

Reply:
Fixed in the current revision (just forced pushed as part of a rebase too)